spring-cloud-aws-messaging/src/main/java/io/awspring/cloud/messaging/config/annotation/SnsConfiguration.java

```diff
@@ -54,4 +60,18 @@ public SnsConfiguration(ObjectProvider<AWSCredentialsProvider> awsCredentialsPro
 				this.regionProvider);
 	}
 
+	@ConditionalOnMissingAmazonClient(SnsMessageManager.class)
+	@Bean
+	public SnsMessageManager snsMessageManager() {
```

Comment:
We cannot use properties to control creation of `SnsMessageManager` not sure what would be best approach here